metaflow/plugins/cards/card_cli.py

```diff
@@ -261,16 +261,24 @@ def render_card(mf_card, task, timeout_value=None):
     type=int,
     help="Maximum amount of time allowed to create card.",
 )
+@click.option(
+    "--with-error-card",
```

Comment:
I would avoid `with` as it has other connotations. Maybe something like `--render-error-card` and `--no-render-error-card`?

Reply:
`--render-error-card` is a flag.